From c0d07c4c6d4f035d052433f1ab1735a7e7fd39a0 Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Fri, 20 Nov 2020 18:32:31 +0100
Subject: [PATCH 3/3] Disable d3d12 requiring fxc.exe

* fxc.exe is not provided by WINE or mingw-w64 and hence not available
  in our build environment
* The version from https://github.com/mozilla/fxc2 builds but it does
  not support all CLI options required by Qt's build system
* The build error looks like this:
  ```
  make[5]: Entering directory '/build/mingw-w64-qt5-declarative/src/qtdeclarative-everywhere-src-5.15.2/build-i686-w64-mingw32-shared/src/plugins/scenegraph/d3d12'
  fxc.exe /nologo /E VS_VertexColor /T vs_5_0 /Fh vs_vertexcolor.hlslh /build/mingw-w64-qt5-declarative/src/qtdeclarative-everywhere-src-5.15.2/src/plugins/scenegraph/d3d12/shaders/vertexcolor.hlsl
  make[5]: fxc.exe: No such file or directory
  make[5]: *** [Makefile.Release:233: vs_vertexcolor.hlslh] Error 127
  ```
---
 src/plugins/scenegraph/scenegraph.pro | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/plugins/scenegraph/scenegraph.pro b/src/plugins/scenegraph/scenegraph.pro
index 39c0c0815c..9cc521a191 100644
--- a/src/plugins/scenegraph/scenegraph.pro
+++ b/src/plugins/scenegraph/scenegraph.pro
@@ -1,5 +1,4 @@
 TEMPLATE = subdirs
 QT_FOR_CONFIG += quick
-qtConfig(d3d12): SUBDIRS += d3d12
 qtConfig(openvg): SUBDIRS += openvg
 
-- 
2.39.0

